gulpfile.js

```diff
@@ -296,10 +296,7 @@ gulp.task('mocha', function (done) {
         error = err;
       })
       .on('end', function () {
-        // When the tests are done, disconnect mongoose and pass the error state back to gulp
-        mongoose.disconnect(function () {
-          done(error);
-        });
+        mongooseService.disconnect(done);
```

Comment:
I almost did this myself as well, but that `error` being passed to gulp's `done` isn't error from mongoose disconnect. It's the error object from Mocha.

Perhaps that variable needs renaming so that it would be clear what's happening here?